### DISCUSSION TITLE
How to Reinstall from Scratch or Revert back to Default Options?

### DISCUSSION THREAD

**Discussion by mcirish** [collin80/SavvyCAN#698] [Q&A]
Dummy me made two preference changes and now SavvyCAN will not run.  I enabled OpenGL and drastically increased the buffer size to something like 4.5 times original.  There are no changed files in the original directory so it's not clear where these user preference changes are stored.  This is a Windows 11 system and I need to get the software back.   

Please advise,   Thanks.

**Comment by Bill400** [comment]
I don't have the answer but ... you may want to reveal/turn on hidden files: Settings/System/For Developers/File Explorer/Show Hidden and System Files

**Comment by mcirish** [comment]
To clarify, the symptom is that NOTHING happens when I try to run SavvyCAN.  No temporary window, no error, no nothing.   Best guess is that one of the program settings I changed are causing an error that prevents the program from starting.  My deep reading is that parameters settings are stored in the Windows registry however numerous attempts at searching for Key terms like SavvyCan, Qsettings, etc have not revealed anything obvious.   As far as I can tell, this compiled software runs out of its' extracted directory location and there is no install or uninstall procedure informing Windows that it is a compliant program.   For now it appears this program is dead to me and I am devastated without it.

**Reply by mcirish** [reply]
This may not be the ideal answer but I was eventually able to restore SavvyCAN functioning in Windows by restoring the operating system files back to an earlier restore point.
